components/atoms/Avatar/avatar-hover-card.tsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable @next/next/no-img-element */
```

Comment:
Eslint is kind of right about that here. Any reason we need to use `img` tag?

Reply:
Yeah, for some reason when using `next/image` the hovercard content won't show up. It does not trigger the hovercard at all :no_mouth:

Reply:
Have you used the priority prop to disable lazy loading? https://nextjs.org/docs/api-reference/next/image#priority

Reply:
Yeah, but still does not work

Reply:
ok, but can you push the code so we can debug this?